scene/gui/rich_text_label.cpp

```diff
@@ -2752,6 +2805,8 @@ void RichTextLabel::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_effects"), &RichTextLabel::get_effects);
 	ClassDB::bind_method(D_METHOD("install_effect", "effect"), &RichTextLabel::install_effect);
 
+	ClassDB::bind_method(D_METHOD("parse_color"), &RichTextLabel::parse_color);
```

Comment:
Please note that in 4.0 it is planned to switch to standard named colors from the `Color` class (#35505).

Reply:
I just pushed a commit which utilizes the changes made to the `Color` class in #35505